pkg/antctl/raw/check/installation/test_podtopodinternode.go

```diff
@@ -27,9 +27,9 @@ func init() {
 	RegisterTest("pod-to-pod-internode-connectivity", &PodToPodInterNodeConnectivityTest{})
 }
 
-func (t *PodToPodInterNodeConnectivityTest) Run(ctx context.Context, testContext *testContext) error {
+func (t *PodToPodInterNodeConnectivityTest) Run(ctx context.Context, testContext *testContext) (error, bool) {
 	if testContext.echoOtherNodePod == nil {
-		return fmt.Errorf("Skipping Inter-Node test because multiple Nodes are not available")
+		return fmt.Errorf("Inter-Node test requires multiple Nodes"), true
```

Comment:
An alternative would be to define a custom error type for skipped tests, but I don't have a strong preference

Reply:
done